homeassistant/components/nanoleaf/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "Nanoleaf",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/nanoleaf",
-  "requirements": ["aionanoleaf==0.2.0"],
+  "requirements": ["aionanoleaf==0.2.1"],
```

Comment:
Note that the retries are hardcoded to 3 instead of using the available parameter.

https://github.com/milanmeu/aionanoleaf/blob/4372e32146bbe6731990d3f649847669807bfb84/aionanoleaf/nanoleaf.py#L76